src/main/java/seedu/address/storage/XmlAdaptedLifeInsurance.java

```diff
@@ -26,31 +26,31 @@
     private String expiryDate;
 
     /**
-     * Constructs an XmlAdaptedPerson.
+     * Constructs an XmlAdaptedLifeInsurance.
      * This is the no-arg constructor that is required by JAXB.
      */
     public XmlAdaptedLifeInsurance() {}
 
 
     /**
-     * Converts a given Person into this class for JAXB use.
+     * Converts a given LifeInsurance into this class for JAXB use.
      *
-     * @param source future changes to this will not affect the created XmlAdaptedPerson
+     * @param source future changes to this will not affect the created XmlAdaptedLifeInsurance
      */
     public XmlAdaptedLifeInsurance(ReadOnlyInsurance source) {
-        owner = source.getOwner().getName().fullName;
-        insured = source.getInsured().getName().fullName;
-        beneficiary = source.getBeneficiary().getName().fullName;
+        owner = source.getOwner().getName();
+        insured = source.getInsured().getName();
+        beneficiary = source.getBeneficiary().getName();
         premium = source.getPremium();
         contractPath = source.getContractPath();
         signingDate = source.getSigningDate();
         expiryDate = source.getExpiryDate();
     }
```

Comment:
need to be careful about this, this returns a Name class that only typecasts into a string because the toString method is defined but since the name class is pretty straightforward so its ok, need to be careful about other classes like Tags cos i messed up on that before

Reply:
Uhm I don't think so? 
Here `source.getOwner()` returns a `InsurancePerson`. `InsurancePerson.getName()` returns a `String`, not a `Name`. This is the class we came up to prevent the case if the person doesn't exist in Lisa.

Reply:
oh i missed that